src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.SortCommandParser;
+import seedu.address.model.Model;
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.person.Person;
+
+/**
+ * Sorts persons in SocialBook according to the specified field.
+ */
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book according to the given parameter.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Sorted all persons";
```

Comment:
If possible can make the message more specific like "Sorted persons by name" or "Sorted persons by priority"

Reply:
Have added this in. Thanks for the idea!